dapps/assets/src/i18n/de.json

```diff
@@ -1,12 +1,45 @@
 {
   "_assets": {
-    "contacts": "Kontakte",
-    "digitaltwins": "Digitale Zwillinge",
-    "digitaltwin": {
-      "digitaltwin-title": "Digitale Zwillinge",
-      "my-own": "Eigene",
+    "contacts": {
+      "add-contact-btn": "Kontakt hinzufügen",
+      "add-contact-desc": "Geben Sie die Evan-ID oder die E-Mail-Adresse Ihres Kontaktes ein um diesen einzuladen.",
```

Comment:
Das wording ist hier etwas ungeschickt, ich denke es ist nicht ganz klar, dass man entweder die Evan ID eines bestehenden Nutzers nehmen sollte, oder die email um einen Nutzer neu einzuladen. Email für bestehenden Nutzer geht ja nicht.

Reply:
Yeah, I find the concept of mixing inviting existing and new accounts troublesome too. But I just followed the wireframes for now.